integrations/ollama/src/ollama_haystack/chat/chat_generator.py

```diff
@@ -0,0 +1,83 @@
+from typing import Any, Dict, List, Optional
+
+import requests
+from haystack import component
+from haystack.dataclasses import ChatMessage
+from requests import Response
+
+
+@component
+class OllamaChatGenerator:
+    """
+    Chat Generator based on Ollama. Ollama is a library for easily running LLMs locally.
+    This component provides an interface to generate text using a LLM running in Ollama.
+    """
+
+    def __init__(
+        self,
+        model: str = "orca-mini",
+        url: str = "http://localhost:11434/api/chat",
+        generation_kwargs: Optional[Dict[str, Any]] = None,
+        template: Optional[str] = None,
+        timeout: int = 30,
```

Comment:
```suggestion
        timeout: int = 120,
```

Experimenting locally with Ollama, I saw that 30 seconds may be too little.
I would propose increasing the default timeout to 120.

Reply:
Do you think it's worth doing on the regular generator a well?